src/index.js

```diff
@@ -76,7 +76,13 @@ const Alpine = {
 
     initializeComponent: function (el) {
         if (! el.__x) {
-            el.__x = new Component(el)
+            try {
+                el.__x = new Component(el)
+            } catch (err) {
```

Comment:
Should we be logging the error? Or are we saying "let the browser log the error but not crash the whole JS thread"

Reply:
I guess throwing it would be at par with what is happening now, but without "crashing" other components.

They are allowed to init before the error is thrown.

Reply:
Yes. If a crashed happens, it still crashes up to the global window.onerror but at least the other components will still initialize.

An alternative design might be to stash the crashes in some Alpine specific location, but since the crashes shouldn’t happen anyway, I like this solution more.

Reply:
Sounds good to me.

Reply:
I like this.